virtcontainers/container.go

```diff
@@ -612,6 +612,24 @@ func (c *Container) unmountHostMounts() error {
 				return err
 			}
 
+			if m.Type == "bind" {
```

Comment:
It would be great if you could add a unit test for this scenario to ` virtcontainers/container_test.go`.

Reply:
Thanks for the suggestion. I'll work on it 😀